eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/RatifySpec.hs

```diff
@@ -223,6 +225,54 @@ paramChangeAffectsProposalsSpec =
       Map.member (GovPurposeId childGai) <$> getParameterChangeProposals `shouldReturn` True
       isDRepAccepted childGai `shouldReturn` False
 
+committeeMinSizeAffectsInFlightProposalsSpec ::
+  forall era.
+  ConwayEraImp era =>
+  SpecWith (ImpTestState era)
+committeeMinSizeAffectsInFlightProposalsSpec =
+  describe "CommitteeMinSize affects in-flight proposals" $ do
+    let setCommitteeMinSize n = modifyPParams $ ppCommitteeMinSizeL .~ n
+        submitATreasuryWithdrawal = do
+          rewardAccount <- registerRewardAccount
+          submitTreasuryWithdrawals [(rewardAccount, Coin 1_000)]
+    it "TreasuryWithdrawal fails to ratify due to an increase in CommitteeMinSize" $ do
+      (drepC, hotCommitteeC, _gpiCC) <- electBasicCommittee
+      setCommitteeMinSize 1
+      gaiPC <-
+        submitParameterChange SNothing $
+          emptyPParamsUpdate
+            & ppuCommitteeMinSizeL .~ SJust 2
+      submitYesVote_ (CommitteeVoter hotCommitteeC) gaiPC
+      submitYesVote_ (DRepVoter drepC) gaiPC
+      treasury <- getsNES $ nesEsL . esAccountStateL . asTreasuryL
+      gaiTW <- submitATreasuryWithdrawal
+      submitYesVote_ (CommitteeVoter hotCommitteeC) gaiTW
+      submitYesVote_ (DRepVoter drepC) gaiTW
```

Comment:
To make it a little more interesting, let's submit the `TreasuryWithdrawal` before the `ParameterChange` proposal